stix_shifter_modules/aws_athena/configuration/lang_en.json

```diff
@@ -55,6 +55,10 @@
                 "label": "AWS IAM Role",
                 "description": "AWS IAM Role is required for only AWS role-based authentication"
             },
+            "aws_assume_role_external_id": {
+                "label": "AWS IAM Role Assume External ID",
```

Comment:
Can we name the label this-
`AWS IAM Role Assume External ID `->  `External ID for AWS IAM Role`